examples/01_basic_io/plot_create_dataset.py

```diff
@@ -38,6 +52,33 @@
         "X7": [9, 8, 7],
     }
 )
+
 dataset = core.Dataset(v="v_alt", X=["X1", "X7"], data=df, add_intercept=False)
 
 pprint(vars(dataset))
+
+###############################################################################
+# Datasets can also contain parallel "datasets"
```

Comment:
I don't like this, but I can't think of a good alternative. Should we just call them "dependent variables"?